src/main/java/seedu/address/logic/commands/DietCommand.java

```diff
@@ -36,7 +35,7 @@
 
     /**
      * @param index of the pet in the filtered pets list to edit the diet
-     * @param diet of the pet to be updated to
+     * @param diet  of the pet to be updated to
```

Comment:
I think there is an extra whitespace here!

Reply:
Thanks for watching out!